meshtastic/tests/test_main.py

```
@@ -4262,3 +4382,207 @@ def test_main_set_ham_empty_string(capsys: pytest.CaptureFixture[str]) -> None:
+
+@pytest.mark.unit
+def test_create_power_meter_requires_initialized_args(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    """_create_power_meter should fail fast if mt_config.args is uninitialized."""
+    monkeypatch.setattr(main_module, "meter", None)
+    monkeypatch.setattr(mt_config, "args", None)
+
+    with pytest.raises(
+        RuntimeError,
+        match="mt_config.args must be initialized before calling _create_power_meter",
```

Ruff (0.15.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)